### PR TITLE
Components Manager

### PR DESCRIPTION
**Pull request by jcakokorev**
Closes gsmainclusivetechlab/interop-test-platform#242

### CODE REVIEW

File: src/app/Http/Controllers/Admin/ComponentController.php [hunk 1]
```diff
@@ -5,15 +5,21 @@
...
 use Illuminate\Support\Facades\DB;
 use Inertia\Inertia;
+use Inertia\Response;
+use Throwable;
 
 class ComponentController extends Controller
 {
     /**
-     * ComponentController constructor.
+     * @return void
```
[the remark this answers is not in the document]
Comment (jcakokorev) [reply src/app/Http/Controllers/Admin/ComponentController.php:22]: @cjol at least not bad

**Comment by jcakokorev** [comment]
@cjol about codacy and doublequotes, as i understand rule called Quotes https://prnt.sc/tv2v6s here https://app.codacy.com/gh/gsmainclusivetechlab/interop-test-platform/patterns/list. And I haven't changed anything in codacy

**Comment by cjol** [comment]
As I understand it, that's just enforcing consistency (so in our case I think that will enforce single quotes because that's what we're primarily using). Did you come across any rule violations recently?

**Comment by jcakokorev** [comment]
Only rule about doublequotes for example here https://app.codacy.com/gh/gsmainclusivetechlab/interop-test-platform/file/47135243742/issues/source?bid=18269119&fileBranchId=18269119 I do not understand. Everything else is ok for me

**Comment by cjol** [comment]
Thanks for pointing out the rule location. I have now changed it from "double" to "single" - so hopefully the next time Codacy runs analysis those issues will disappear.

**Pull request merged**